roles/gateway/README.rst

```diff
@@ -0,0 +1,77 @@
+gateway
+=======
+
+This role can be used to set up a new OpenVPN gateway for a Ceph test lab 
+as well as maintain user access provided a secrets repo is configured.
+
+This role supports CentOS 7.2 only at this time.  It's current intended use
+is to maintain the existing OpenVPN gateway in our Sepia_ lab.
+
+It does the following:
+- Installs and updates necessary packages
+- Maintains user list
+
+Prerequisites
++++++++++++++
+
+- CentOS 7.2
+
+Variables
++++++++++
+
+A list of packages to install that is specific to the role.  The list is defined in the var file in ``vars/``::
+
+    packages: []
+
+The directory in which the OpenVPN server CA, keys, certs, and user file should be saved.  Defined in the secrets repo::
+
+    openvpn_data_dir: []
+
+
+Contains paths, file permission (modes), and data to store and maintain OpenVPN CA, cert, and key.  Defined in the secrets repo::
+
+    gateway_secrets: []
+
+    # Example:
+    gateway_secrets:
+      - path: "{{ openvpn_data_dir }}/server.crt"
+        mode: 0644
+        data: |
+          -----BEGIN CERTIFICATE-----
+          etc...
+
+A list of users that don't have their ssh pubkey added to the ``teuthology_user`` authorized_keys but still need VPN access::
+
+    openvpn_users: []
```

Comment:
Is this in addition to some other list of users that get both?  If so, talk about how those interact, and if so, maybe call this openvpn_only_users or something